capacitor.config.ts

```diff
@@ -24,7 +24,7 @@ const config: CapacitorConfig = {
   webDir: 'build',
   ...serverConfig,
   ios: {
-    backgroundColor: '000000',
```

Comment:
Remove this `backgroundColor` is causing a white flash on launch. Can we retain this, or does it interfere with the solution?

Even if it does not change with the theme, it would be better to have a black flash than a white flash.

Reply:
Good idea.